forecast/paper/response-to-reviewers.tex

```diff
@@ -0,0 +1,304 @@
+\end{resp}
+
+
+
+  4. Despite the fact that a detailed description of terms in the text is
+  impossible, it would I think be helpful to say if there is any significance to
+  the names they associate with their first 2 and 4th indicators.
+
+
+\begin{resp}
+  It seems that the Reviewer is referring to CHNG-COVID and DV-CLI. These are
+  described in the Methods Section, Signals and Locations Subsection. To be more
+  specific, Change 
+  Healthcare is a large healthcare insurance claims processor. The difference
+  between ``-CLI'' and ``-COVID'' is that the first measures insurance
+  claims related to symptoms
+  consistent with COVID while the second requires a medical diagnosis (through
+  testing or presumed positive by the medical provider). The ``DV'' modifier
+  corresponds to different claims processors (not Change Healthcare) who wish to
+  remain anonymous.
+
+  TODO Do we want to alter the text at all?
+\end{resp}
+
+
+
+  5. What are Change Healthcare claims..as opposed to others?
+
+
+\begin{resp}
+  Hopefully the discussion to point 4 above has cleared up any confusion.
+\end{resp}
+
+
+\section*{Reviewer 2}
+\begin{quote}
+  \emph{(items numbered by the editor for convenience of reference)}
+\end{quote}
+
+\subsection*{Statistical or Methodological Comments:}
+  
+1. Is this really a stationary phenomenon? Over the timerange extending from
+pre-lockdown to today, it certainly isn't. The method in this paper of
+presenting model prediction errors makes it impossible to see this. People in
+econometrics look at cumsums of normalized prediction errors. You can see change
+points and trends very easily.
+
+\begin{resp}
+  There is little reason to believe that this data is stationary. The models
+  however are trained and evaluated over reasonably short periods (2-3 months),
+  over which time the behavior may well be stationary. We have added a section
+  to the appendix that uses the cumulative mean as suggested (the sum is
+  difficult to see due to the scaling). With some exceptions at the beginning of
+  the evaluation period, the relative ordering remains fairly consistent. The AR
+  model is easily the worst.
+
+  TODO I have added the code and a caption for the figure to the Supplement, but
+  no corresponding 
+  text. It seems a fairly minor addition, not necessarily deserving its own
+  section. 
+\end{resp}
+
+2. Are the results in this paper statistically significant? I see no discussion
+of this question, which seems unbelievable for me to be saying, given the
+authorship. And yet here I am.
+
+\begin{resp}
+  This is an important point that we should have addressed more fully.
+  Conventional methods for assessing statistical significance that we explored
+  were overconfident, to the point of being unbelievable. To a large degree,
+  this is a ``huge n, tiny p'' problem so even bootstrap analyses result in
+  extremely narrow intervals. Essentially, ``everything is significant''.
```

Comment:
Isn't spatiotemporal correlation a culprit here... i.e., the effective n is much smaller?

Reply:
(revise this)